src/main/java/seedu/address/logic/commands/LessonDeleteCommand.java

```diff
@@ -0,0 +1,125 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Fee;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Remark;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Deletes a Lesson from an existing person in the address book.
+ */
+public class LessonDeleteCommand extends Command {
+    public static final String COMMAND_WORD = "ldelete";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes the lesson identified by lesson index"
+        + " of the person identified by the index number used in the displayed person list.\n"
+        + "Parameters: INDEX (must be a positive integer) LESSON_INDEX (must be a positive integer)\n"
+        + "Example: " + COMMAND_WORD + " 1 " + "1";
+
+    public static final String MESSAGE_DELETE_LESSON_SUCCESS = "Deleted Lesson: %1$s\nfor Person: %2$s";
```

Comment:
eh not sure about this but i thought now our message should all be students instead of person if it is not then should we change it to students? since our UG and DG calls them students